app/views/annotations/_form.html.erb

```diff
@@ -0,0 +1,151 @@
+<div id="annotation-form">
+  <%= form_with model: @annotation do |f| %>
+    <b><%= t('admin.annotation.time') %></b> &nbsp <%= TimeStamp.new(total_seconds: @total_seconds).hms_colon_string %>
+
+    <br><br>
+
+    <!--- HIDDEN FIELDS -->
+    <%= f.hidden_field :medium_id, value: @medium_id %>
+    <%= f.hidden_field :total_seconds, value: @total_seconds %>
+    <div hidden>
+      <div id="annotation_id">
+        <%= @annotation.id %>
+      </div>
+       <!--- for prefilling the edit form -->
+      <div id="annotation_subtext">
+        <%= @annotation.subtext %>
+      </div>
+    </div>
+
+    <!--- COMMENT TEXT AREA -->
+    <b><%= t('admin.annotation.comment') %></b>
+    <%= f.text_area :comment, class: 'form-control' %>
+
+    <br>
+
+    <!--- PREVIEW TOGGLE -->
+    <div id="preview-toggle">
+      <label class="switch">
+        <input id="preview-toggle-check" type="checkbox">
+        <span class="slider round"></span>
+      </label>
+      &nbsp
+      <%= t('admin.remark.preview') %>
+    </div>
+
+    <br>
+
+    <!--- COLOR SELECTION -->
+    <b><%= t('admin.annotation.color') %></b>
+    <div id="annotation-color-picker">
+      <% for i in 1..15 do %>
+        <%= f.radio_button :color,
+                           annotation_color(i),
+                           id: "annotation_color#{i}" %>
```

Comment:
Why do we have the `annotation_color(int)` in the `AnnotationsHelper` in the first place? The only usage I see is here. Why not add a class here, so that we only need the map of colors defined in `thyme.scss`? (where I feel it is perfectly placed as this is frontend not backend related)